Xamarin.Forms.Platform.Android/Extensions/TextViewExtensions.cs

```diff
@@ -151,5 +153,35 @@ public static void RecalculateSpanPositions(this TextView textView, Label elemen
 				((ISpatialElement)span).Region = Region.FromLines(lineHeights, labelWidth, startX, endX, yaxis).Inflate(10);
 			}
 		}
+
+		public static void SetAutoFitMode(this TextView textView, Label label)
+		{
+			if ((int)Forms.SdkInt < 14)
+				return;
+
+
+			switch (label.AutoFitText)
+			{
+				case AutoFitTextMode.FitToWidth:
+					textView.SetMaxLines(1);
```

Comment:
Should this call SetValueFromRenderer to change MaxLines property on the label and trigger a property changed just-in case someone subscribed to two-way binding? 🤔

Reply:
Thanks! yeah, I think this is a good point!

Reply:
Done!